tests/test_netstats.py

```diff
@@ -16,6 +16,31 @@
 logger.setLevel(50)
 
 
+@pytest.mark.parametrize("value", [True, 3.14, "word", {"project":"PyNets"}])
+def test_get_prop_type(value):
+    """
+    Test for get_prop_type() functionality
+    """
+    start_time = time.time()
+    tname, value, key = netstats.get_prop_type(value)
+    print("%s%s%s" % ('thresh_and_fit (Functional, proportional thresholding) --> finished: ',
+                      np.round(time.time() - start_time, 1), 's'))
+    assert tname is not None
+
+def test_nx2gt():
+    """
+    Test for nx2gt() functionality
+    """
+    base_dir = str(Path(__file__).parent/"examples")
+    in_mat = np.load(f"{base_dir}/miscellaneous/graphs/002_modality-func_rsn-Default_model-cov_nodetype-spheres-2mm_smooth-2fwhm_hpass-0.1Hz_thrtype-PROP_thr-0.95.npy")
+    nxG = nx.from_numpy_array(in_mat)
+
+    start_time = time.time()
+    gtG = netstats.nx2gt(nxG)
+    print("%s%s%s" % ('thresh_and_fit (Functional, proportional thresholding) --> finished: ',
+                      np.round(time.time() - start_time, 1), 's'))
+    assert gtG is not None
```

Comment:
Here too, we should expand upon this assert. One thing to consider is that if nxG is not None (i.e. and it's a true nx.Graph versus, say, a np.ndarray), then gtG should always return a valid graph_tool graph object.